src/common/perf_counters.h

```diff
@@ -278,7 +279,7 @@ class PerfCounters
   PerfCounters(const PerfCounters &rhs);
   PerfCounters& operator=(const PerfCounters &rhs);
   void dump_formatted_generic(ceph::Formatter *f, bool schema, bool histograms,
-                              const std::string &counter = "") const;
+                              bool dump_labeled = false, const std::string &counter = "") const;
```

Comment:
Defaulting leftover and this line is too long too:
```suggestion
                              bool dump_labeled,
                              const std::string &counter = "") const;
```